static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -47,6 +47,11 @@ export type OnboardingLayoutProps = {
   newOrg?: boolean;
 };
 
+export interface FeatureFlagOnboardingLayoutProps extends OnboardingLayoutProps {
+  integration?: string;
+  provider?: string;
+}
```

Comment:
One last nit: would move this into `static/app/components/events/featureFlags/featureFlagOnboardingLayout.tsx`.
As normally our props interfaces are in the same file as the component.